SemRush issues July 2022 - meta-descriptions v3 (#5453)
wazuh/wazuh-documentation: source/deployment-options/splunk/splunk-wazuh.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>    :description: Learn how to install Elastic Stack for using Wazuh on Debian

Reviewer: ```suggestion
   :description: Install the Wazuh manager for a Wazuh with Splunk installation
```
